python/ray/dataframe/__init__.py

```diff
@@ -28,14 +28,11 @@ def get_npartitions():
 # We import these file after above two function
 # because they depend on npartitions.
 from .dataframe import DataFrame  # noqa: 402
-from .dataframe import from_pandas  # noqa: 402
```

Comment:
doesn't need to be addressed in this PR, but instead of putting `# noqa: 402`, we should probably just fix the linting error. Is this "no relative imports"? If so, it can be fixed by doing something like `from ray.dataframe import DataFrame`

Reply:
It fails because the `npartitions` needs to be defined first based on the current structure. I can fix this in a future PR.